bsp/qemu-sifive-e31/core.dts

```diff
@@ -170,7 +174,7 @@
                         reg = <0x10014000 0x1000 0x20000000 0x20000000>;
                         reg-names = "control", "mem";
                         clocks = <&hfclk>;
-                        pinmux = <&gpio0 0x0003C 0x0003C>;
+                        pinmux = <&gpio0 0x0 0x0003C>;
```

Comment:
same here, previously incorrect?

Reply:
dts values changed, as per a change in gpio driver.
https://github.com/sifive/freedom-metal/pull/232